test/Microsoft.AspNet.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -1689,7 +1731,7 @@ private class Person11
             Assert.Null(entry.RawValue);
             Assert.Null(entry.AttemptedValue);
             var error = Assert.Single(modelState["Customer.Name"].Errors);
-            Assert.Equal("A value for the 'Name' property was not provided.", error.ErrorMessage);
+            Assert.Equal("A value for the 'Customer.Name' property was not provided.", error.ErrorMessage);
```

Comment:
This illustrates the utility of the full path better than all of the tests with prefixes and custom binder names.